misc/python/materialize/parallel_workload/database.py

```diff
@@ -417,34 +406,21 @@ def __init__(
         self.view_id = len(self.views)
         self.roles = [Role(i) for i in range(rng.randint(0, MAX_INITIAL_ROLES))]
         self.role_id = len(self.roles)
-        self.compute_clusters = [
-            Cluster(
-                ClusterType.COMPUTE,
-                i,
-                managed=rng.choice([True, False]),
-                size=rng.choice(["1", "2", "4"]),
-                replication_factor=rng.choice([1, 2, 4, 5]),
-                introspection_interval=rng.choice(["0", "1s", "10s"]),
-            )
-            for i in range(rng.randint(0, MAX_INITIAL_COMPUTE_CLUSTERS))
-        ]
-        self.compute_cluster_id = len(self.compute_clusters)
         # At least one storage cluster required for WebhookSources
-        self.storage_clusters = [
+        self.clusters = [
             Cluster(
-                ClusterType.STORAGE,
                 i,
                 managed=rng.choice([True, False]),
                 size=rng.choice(["1", "2", "4"]),
-                replication_factor=1,  # cannot create source in cluster with more than one replica
```

Comment:
This is still true -- the limitation of sources and the replication factor has not been lifted as far as I know.

Reply:
Yes, but I check this in action.py now when creating new sources, so that they are only created on a fitting cluster.